src/applications/vaos/referral-appointments/ConfirmApprovedPage.jsx

```diff
@@ -0,0 +1,116 @@
+import React, { useState, useEffect } from 'react';
+import PageLayout from '../appointment-list/components/PageLayout';
+
+const staticData = {
+  typeOfCare: 'Primary Care',
+  provider: 'Kristina Jones, NP',
+  clinic: 'Tender Care Clinic',
+  address: '1234 Cool st.',
+  city: 'Aewsometown, AZ',
+  phone: '555-123-4567',
+  date: 'Thursday, June 20, 2024',
+  time: '4:15 p.m. PST',
+  details: 'Back pain',
+};
+
+const dynamicData = {
+  // Fill in real data here, network call, etc.
+};
+
+const getData = (typeOfData = 'static') => {
+  return typeOfData === 'static' ? staticData : dynamicData;
+};
+
+export default function ConfirmApproved() {
+  const [confirmedData, setConfirmedData] = useState(0);
+
+  // on react component mount, fetch data
+  useEffect(() => {
+    setConfirmedData(getData());
+  }, []);
+
+  return (
+    <PageLayout showBreadcrumbs showNeedHelp>
```

Comment:
I noticed the width of the `ConfirmApprovedPage` & `ReviewApproved` pages are larger than what we have across the appointments application.
 
**New Referral Page**: 

![Screenshot 2024-05-20 at 12 00 11 PM](https://github.com/department-of-veterans-affairs/vets-website/assets/47654119/29caa83b-ac91-4aa3-b717-997a01c9fe5a)

**Existing page:** 

![Screenshot 2024-05-20 at 12 01 03 PM](https://github.com/department-of-veterans-affairs/vets-website/assets/47654119/8676b2ea-7626-402c-9565-d7a659b9a268)

We have a `FormLayout` component that we can use instead of the `PageLayout` component. This component handles styling and adds the **New Appointment** eyebrow too.

`import FormLayout from '../new-appointment/components/FormLayout';`

```suggestion
     <FormLayout>
      <h1>Review your appointment details</h1>
      ...
</FormLayout>
```